var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -154,6 +154,10 @@ def patch(self):
         # https://github.com/Perl/perl5/issues/15544 long PATH(>1000 chars) fails a test
         os.chmod("lib/perlbug.t", 0o644)
         filter_file("!/$B/", "! (/(?:$B|PATH)/)", "lib/perlbug.t")
+        # Several non-existent flags cause Intel@19.1.3.304 to fail
+        with when("%intel@19.1.3.304"):
```

Comment:
Maybe the condition on the version should be relaxed a bit, we use 19.1.3 here for example.

Reply:
Makes sense. Just changed it to 19.1.3.